tests/test_server.py

```diff
@@ -207,9 +207,13 @@ def send_request() -> None:
     train_request.terminate()
 
 
-# due to unknown reasons this test can not be run in pycharm, it
-# results in segfaults...will skip in that case - test will still get run on CI
+# Due to unknown reasons this test can not be run in pycharm, it
+# results in segfaults...will skip in that case - test will still get run on CI.
+# It also doesn't run on Windows because of Process-related calls and an attempt
```

Comment:
should we create an issue and reference it here in the comment?

Reply:
done!